src/libraries/System.Private.CoreLib/src/System/Numerics/Vector.cs

```diff
@@ -900,46 +765,40 @@ public static Vector<double> Floor(Vector<double> value)
             return (Vector<ulong>)value;
         }
 
-
-        /// <summary>
-        /// Reinterprets the bits of the given vector into those of a vector of signed 64-bit integers.
-        /// </summary>
-        /// <param name="value">The source vector</param>
+        /// <summary>Reinterprets the bits of a specified vector into those of a vector of long integers.</summary>
+        /// <param name="value">The source vector.</param>
+        /// <typeparam name="T">The vector type. <c>T</c> can be any primitive numeric type.</typeparam>
         /// <returns>The reinterpreted vector.</returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static Vector<long> AsVectorInt64<T>(Vector<T> value) where T : struct
         {
             return (Vector<long>)value;
         }
 
-        /// <summary>
-        /// Reinterprets the bits of the given vector into those of a vector of 32-bit floating point numbers.
-        /// </summary>
-        /// <param name="value">The source vector</param>
+        /// <summary>Reinterprets the bits of a specified vector into those of a single-precision floating-point vector.</summary>
+        /// <param name="value">The source vector.</param>
+        /// <typeparam name="T">The vector type. <c>T</c> can be any primitive numeric type.</typeparam>
         /// <returns>The reinterpreted vector.</returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static Vector<float> AsVectorSingle<T>(Vector<T> value) where T : struct
         {
             return (Vector<float>)value;
         }
 
-        /// <summary>
-        /// Reinterprets the bits of the given vector into those of a vector of 64-bit floating point numbers.
-        /// </summary>
-        /// <param name="value">The source vector</param>
+        /// <summary>Reinterprets the bits of a specified vector into those of a double-precision floating-point vector.</summary>
+        /// <param name="value">The source vector.</param>
+        /// <typeparam name="T">The vector type. <c>T</c> can be any primitive numeric type.</typeparam>
         /// <returns>The reinterpreted vector.</returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static Vector<double> AsVectorDouble<T>(Vector<T> value) where T : struct
         {
             return (Vector<double>)value;
         }
 
-        /// <summary>
-        /// Widens a Vector{Byte} into two Vector{UInt16}'s.
+        /// <summary>Widens a <c>Vector&lt;Byte&gt;</c> into two <c>Vector&lt;UInt16&gt;</c> instances.</summary>
```

Comment:
There are a bunch of `&lt;` and `&gt;` contents below here. Should these be encoded differently?

Reply:
Good catch. These should have been mentioned  as <`see cref="docid" />`.

Reply:
Note: This is also a case of something that came as is from MS Docs. I'll fix it manually here, so next time we sent these docs to MS Docs, they'll get the correct content.

Reply:
This particular case is referring to the type of the passed parameters, which is `Vector<byte>`. It seems I cannot mention them as a `see cref`, which is why these comments mention them this way.

I'll leave these specific cases unmodified, but I did find a few other similar cases in the same file that can be fixed.

Reply:
> It seems I cannot mention them as a see cref, which is why these comments were added this way.

You can, as `cref="Vector{Byte}" />`, the issue is that somewhere between Roslyn and msdocs processing the reference, this is reverted to simply `Vector{T}` and we lose the generic type information.

Reply:
Yep, I found that option to be ugly. I prefer to keep the generic type, so the `<c></c>` seems to be the most appropriate workaround.